components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/Scope.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.common.model;
+
+/**
+ * Represents a scope.
+ */
+public class Scope {
+
+    private String id;
+    private String name;
+    private String displayName;
+    private String description;
+    private String apiID;
+    private String orgID;
+
+    public Scope() {
+
+    }
+
+    public Scope(String scopeId, String scopeQualifiedName, String scopeDisplayName, String scopeDescription) {
+
+        this.id = scopeId;
+        this.name = scopeQualifiedName;
+        this.displayName = scopeDisplayName;
+        this.description = scopeDescription;
+    }
+
+    public Scope(String scopeId, String scopeQualifiedName, String scopeDisplayName, String scopeDescription,
+                 String apiID, String orgID) {
+
+        this.id = scopeId;
+        this.name = scopeQualifiedName;
+        this.displayName = scopeDisplayName;
+        this.description = scopeDescription;
+        this.apiID = apiID;
+        this.orgID = orgID;
+    }
+
+    public String getId() {
```

Comment:
add new line after the method signature. Check other places as well

Reply:
https://github.com/wso2/carbon-identity-framework/pull/4854/commits/8dd16447d6000412e3ff2dd1b2a9ed78249f2d13